server/config/config.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Flow Playground
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+	"log"
+)
+
+// config holds all parsed environment variables
+var config struct {
```

Comment:
Haha really? It just initializes a single struct named config. Instead of making it a type and then having to initialize it separately. Kind of useful if you only need one struct.